src/workflow/languageServer.ts

```diff
@@ -26,6 +27,7 @@ export async function initLanguageServer(context: vscode.ExtensionContext) {
   const initializationOptions: InitializationOptions = {
     sessionToken: session?.accessToken,
     userAgent: userAgent,
+    githubApiUrl: isUseEnterprise() ? getGitHubApiUri() : undefined,
```

Comment:
```suggestion
    githubApiUrl: isUseEnterprise() && getGitHubApiUri(),
```

Reply:
I might be wrong, but wouldn't this set githubApiUrl to `false` if `isUseEnterprise()` returns false?

Reply:
Definitely you are right, since `isUseEnterprise()` returns a boolean (sorry for the bad suggestion)